HelpfulItem.h

```diff
@@ -1,15 +1,12 @@
 #pragma once
 
 #include "Item.h"
-#include "Dwarf.h"
 #include "Character.h"
-#include "Paladin.h"
-#include "DragonSlayer.h"
-#include "Dragon.h"
+#include "Utility.h"
```

Comment:
Don't include what you don't use
no member functions or member variables in `Character` are being used in this header file
- [ ] revise